docker-compose.yml

```diff
@@ -7,4 +7,6 @@ services:
       - "8080:8080"
     environment:
       - USER=vagrant
+    volumes:
+      - ./:/app
```

Comment:
Mounts the source code directory as a volume under `/app` for local development.